src/lib/bind/effects.html

```diff
@@ -77,7 +77,7 @@
       if (args) {
         var fn = this[effect.method];
         if (fn) {
-          this[effect.property] = fn.apply(this, args);
+          this._setProperty(effect.property, fn.apply(this, args));
```

Comment:
Cannot make a readOnly property named `property`. Perhaps we should change the name of `_setProperty`?